ui/DesignSystem/Component/Stat.svelte

```diff
@@ -0,0 +1,19 @@
+<script>
+  import { Numeric } from "../Primitive";
+
+  export let count = null;
+  export let icon = null;
+  export let style = null;
+</script>
+
+<style>
+  .stat {
+    display: flex;
+    align-items: center;
+  }
+</style>
+
+<div class="stat" {style}>
+  <svelte:component this={icon} style="margin-right: 4px;" />
+  <Numeric style="color: var(--color-gray);">{count}</Numeric>
```

Comment:
The number should be darkgray (icon is gray):

<img width="313" alt="Screenshot 2020-02-27 at 14 22 20" src="https://user-images.githubusercontent.com/158411/75449086-ace12e00-596c-11ea-950f-d0b0380e0302.png">